resources/definitions/fokoos_base.def.json

```
@@ -0,0 +1,215 @@
+        
+        "machine_max_feedrate_x": { "value": 500 },
+        "machine_max_feedrate_y": { "value": 500 },
+        "machine_max_feedrate_z": { "value": 10 },
+        "machine_max_feedrate_e": { "value": 60 },
+
+        "machine_max_acceleration_x": { "value": 500 },
+        "machine_max_acceleration_y": { "value": 500 },
+        "machine_max_acceleration_z": { "value": 100 },
+        "machine_max_acceleration_e": { "value": 5000 },
+        "machine_acceleration": { "value": 500 },
+
+        "machine_max_jerk_xy": { "value": 10 },
+        "machine_max_jerk_z": { "value": 0.4 },
+        "machine_max_jerk_e": { "value": 5 },
+
+        "machine_heated_bed": { "default_value": true },
+        "material_diameter": { "default_value": 1.75 },
+
+
+        "line_width": { "value": "machine_nozzle_size" },
+
+
+        "wall_thickness": {"value": "line_width * 2" },
+        "optimize_wall_printing_order": { "value": true },
+        "fill_outline_gaps": { "value": true },
+        "z_seam_type": { "value": "'back'" },
+        "z_seam_corner": { "value": "'z_seam_corner_weighted'" },
+        "wall_0_wipe_dist": { "value": 0.0 },
+
+
+        "top_bottom_thickness": {"value":  "layer_height_0 + layer_height * 3" },
+        "skin_overlap": { "value": 10.0 },
+
+        "infill_pattern": { "value": "'lines' if infill_sparse_density > 50 else 'cubic'" },
+        "infill_before_walls": { "value": false },
+        "infill_overlap": { "value": 30.0 },
+        "infill_wipe_dist": { "value": 0.0 },
+
+
+        "speed_print": { "value": 50.0 } ,
+        "speed_infill": { "value": "speed_print" },
+        "speed_wall": { "value": "speed_print / 2" },
+        "speed_wall_0": { "value": "speed_wall" },
+        "speed_wall_x": { "value": "speed_wall" },
+        "speed_topbottom": { "value": "speed_print / 2" },
+        "speed_roofing": { "value": "speed_topbottom" },
+    
+        "speed_layer_0": { "value": 20.0 },
+        "speed_print_layer_0": { "value": "speed_layer_0" },
+        "speed_travel": { "value": "150.0 if speed_print < 60 else 250.0 if speed_print > 100 else speed_print * 2.5" },
+        "speed_travel_layer_0": { "value": "100 if speed_layer_0 < 20 else 150 if speed_layer_0 > 30 else speed_layer_0 * 5" },
+        "skirt_brim_speed": { "value": "speed_layer_0" },
+
+
+        "acceleration_enabled": { "value": false },
+        "acceleration_print": { "value": 500 },
+        "acceleration_travel": { "value": 500 },
+        "acceleration_travel_layer_0": { "value": "acceleration_travel" },
+        "acceleration_roofing": { "enabled": "acceleration_enabled and roofing_layer_count > 0 and top_layers > 0" },
+
+        "jerk_enabled": { "value": false },
+        "jerk_print": { "value": 8 },
+        "jerk_travel": { "value": "jerk_print" },
+        "jerk_travel_layer_0": { "value": "jerk_travel" },
+
+
+
+        "retraction_enable": { "value": true },
```

```suggestion
        "retraction_enable": { "value": true },
```
This overrides the default with the same value. As a rule of thumb, you should try to adhere to the DRY (Don't Repeat Yourself) principle. This will make it a lot easier to maintain profiles in the future!

---

Yes,DRY.